### PR TITLE
Email template changes

### PR DESCRIPTION
**Pull request by npawar**
This pull request contains the changes for the email templates fom the backend and the frontend. It has 3 types of templates: 1) Single anomaly - with an anomaly card and the screenshot 2) Multiple anomalies - a tabular format, with links to investigate each anomaly, or all at once 3) Daily anomaly report - a tabular format, with a summary on top, and links to investigate each one

### CODE REVIEW

**Comment by jfim** [comment]
Please limit the length of the pull request titles to 50 characters.

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java [hunk 1]
```diff
@@ -1,42 +1,57 @@
...
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class AnomalyReportGenerator {
 
   private static final Logger LOG = LoggerFactory.getLogger(AnomalyReportGenerator.class);
 
   private static final AnomalyReportGenerator INSTANCE = new AnomalyReportGenerator();
+  private static final String DATE_PATTERN = "MMM dd, hh:mm a";
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java]: Date pattern to hh to HH and a to z or include that time zone in the email

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java [hunk 2]
```diff
@@ -122,15 +141,21 @@ public void buildReport(long startTime, long endTime, List<MergedAnomalyResultDT
...
             anomaly.getFeedback() == null ? "Not Resolved" : "Resolved(" + anomaly.getFeedback().getFeedbackType().name() + ")");
 
+        DateTimeZone dateTimeZone = Utils.getDataTimeZone(anomaly.getCollection());
         AnomalyReportDTO anomalyReportDTO = new AnomalyReportDTO(String.valueOf(anomaly.getId()),
             getAnomalyURL(anomaly, configuration.getDashboardHost()),
-            String.valueOf(anomaly.getAvgBaselineVal()),
-            String.valueOf(anomaly.getAvgCurrentVal()),
-            anomaly.getDimensions().toString(),
-            String.format("%.2f", getTimeDiffInHours(anomaly.getStartTime(), anomaly.getEndTime())), // duration
+            String.format("%.2f", anomaly.getAvgBaselineVal()),
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java]: use  a pattern that avoids .00 if value is integer

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java [hunk 3]
```diff
@@ -122,15 +141,21 @@ public void buildReport(long startTime, long endTime, List<MergedAnomalyResultDT
...
+            String.format("%.2f", anomaly.getAvgCurrentVal()),
+            getDimensionsString(anomaly.getDimensions()),
+            String.format("%.2f hours (%s to %s) %s",
+                getTimeDiffInHours(anomaly.getStartTime(), anomaly.getEndTime()),
```
Comment (kishoreg) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java]: Why always in Hours? I think there is nice method in DateTime that gives duration in readable form

**Pull request merged**